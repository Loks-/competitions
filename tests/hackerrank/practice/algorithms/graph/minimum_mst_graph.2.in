100
2 1 1
2 1 18
4 5 9
10 30 18
9 23 20
10 45 19
10 45 20
10 45 10
10 45 11
10 45 13
10 45 14
10 45 15
10 45 17
10 45 18
10 44 15
10 44 16
10 44 20
10 42 9
10 42 10
10 42 11
10 42 12
10 42 13
10 42 14
10 39 17
8 24 14
8 24 15
8 24 16
10 39 18
10 39 19
10 39 20
9 34 15
9 34 16
9 34 18
9 34 19
9 32 17
7 20 18
7 17 13
9 32 20
9 33 8
9 33 9
9 33 10
9 33 11
9 33 12
8 25 16
8 25 18
8 25 19
8 25 20
7 17 17
5 6 20
6 14 7
2 1 20
3 2 2
3 2 17
3 3 14
3 3 15
3 3 16
3 3 19
3 3 20
4 5 15
4 5 16
4 5 17
4 5 18
4 5 19
4 5 20
5 7 7
5 7 8
5 7 9
5 7 10
5 7 11
5 8 14
5 8 15
5 8 16
5 8 17
5 8 18
5 8 19
5 10 17
5 10 18
5 10 19
5 10 20
6 11 12
6 11 13
6 11 16
6 13 10
6 13 11
6 13 12
6 13 13
6 13 14
6 13 15
6 13 16
6 13 17
6 13 18
10 28 17
9 16 19
6 15 19
6 15 20
6 15 16
6 14 18
6 14 19
8 15 17
7 18 19
