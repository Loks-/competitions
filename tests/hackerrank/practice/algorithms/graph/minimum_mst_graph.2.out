1
18
15
39
35
99
107
54
62
75
80
84
89
90
82
85
104
42
48
54
60
66
72
63
45
48
51
66
69
72
67
68
80
86
68
60
31
80
33
38
43
48
53
58
66
70
74
39
22
22
20
2
17
21
23
24
29
30
25
27
29
30
32
34
10
11
12
13
14
28
30
32
34
36
38
44
47
49
50
18
19
22
26
29
32
35
38
39
42
45
48
36
27
59
60
50
53
55
25
57
